src/main/webapp/assets/modules/get-promoted/userPromotionJobInfoDirective.js

```diff
@@ -0,0 +1,38 @@
+techlooper.directive("userPromotionJobInfo", function ($http, userPromotionService, $location, utils, $translate) {
+  return {
+    restrict: "E",
+    replace: true,
+    templateUrl: "modules/get-promoted/userPromotionJobInfo.html",
+    link: function (scope, element, attr, ctrl) {
```

Comment:
'ctrl' is defined but never used.<br>'attr' is defined but never used.<br>'element' is defined but never used.